### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN004.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#50] [General]
# https://blog.3place.cn/3place/lmyn/LMYN004.html

https://blog.3place.cn/3place/lmyn/LMYN004.html

**Comment by picklecai** [comment]
又概述了一遍。

**Comment by picklecai** [comment]
这完美的牟尼啜了点水，
站在那里，双手合十，
坐在东向的达哩薄草上，
用达磨来检查全部故事。 

带上脑子，再纵观一遍，整体思考一遍。
达磨就是上面说到的神智。

**Comment by picklecai** [comment]
蚁垤的过程：
1. 听故事
2. 受感动
3. 被（眼前猎人杀鸟的暴行）触动
4. 发明了一种文体
5. 用新文体重述故事（这是神/别人的建议）
6. 重述之前，思考一下这个故事是否值得、是否合适这样处理。（这是自己的思考）

1、2是内容缘起（有罗摩的故事）。
3、4是文体缘起（有了输洛迦）。
5、6是内容和文体的结合缘起，也就是整个《罗摩衍那》的缘起。

“用达磨来检查全部故事。 ”
想来最后也是“增删十载、九易其稿、字字是血”。起头的时候，总得想想在这个主题上花精力是否合适。
